context.go

```diff
@@ -380,13 +404,16 @@ func (c *Context) Sqrt(d, x *Decimal) (Condition, error) {
 		ed.Add(approx, approx, New(819, -4))
 	}
 
+	// TODO(mjibson): Now we repeatedly improve approx. Our precision improves
```

Comment:
I'm not convinced it's best to add comments like this, that are describing how or why the paper chose this algorithm. I think those descriptions are best left in the paper. I think comments in our implementation should focus on differences between us and the paper, or how we do things in Go that look slightly different than the paper.